scripts/init.sh

```diff
@@ -24,10 +24,34 @@ for var in "${vars_to_check[@]}"; do
   export "${var}"
 done
 
+# Setting the Node Role
+env_file_exist "${ROOT_DIR}/${ENV_FILE}"
+# shellcheck disable=SC1090
+source "${ROOT_DIR}/${ENV_FILE}" || { echo "Error: could not source ${ROOT_DIR}/${ENV_FILE} file. Fix it before proceeding any further.  Exiting..."; exit 1; }
+
+if [ -z "${SCNODE_ROLE:-}" ]; then
+  read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
+  SCNODE_ROLE="${SCNODE_ROLE_VALUE}"
```

Comment:
I think you do not need this step here. Just keep the valu under `scnode_role_value` and in the end use this var under your sed commands